site/docs/streams/streams.md

```diff
@@ -4,7 +4,7 @@ title:  "Streams"
 number: 2
 ---
 
-# Streams (experimental)
```

Comment:
Definitely no longer "experimental" :)

Reply:
I have no idea how to gauge this.

Reply:
By this I mean we use at work at a large scale and so far it's solid.

Reply:
ok, good for me